rclrs/src/node/mod.rs

```diff
@@ -312,3 +306,62 @@ impl Node {
         domain_id
     }
 }
+
+impl<'a> NodeBuilder<'a> {
+    /// Creates new builder for Node.
+    pub fn new(name: &'a str, context: &'a Context) -> NodeBuilder<'a> {
+        let mut default_domain_id = 0;
+        // SAFETY: No preconditions for this function.
+        let ret = unsafe { rcl_get_default_domain_id(&mut default_domain_id) };
+        debug_assert_eq!(ret, 0);
+
```

Comment:
These lines about the domain id seem to be currently unused, correct?

Reply:
I forgot that. I'll fix it.